Base/usr/share/man/man7/man.md

```diff
@@ -0,0 +1,57 @@
+## Name
+
+man - SerenityOS manual system
+
+## Description
+
+The SerenityOS manual pages, or "man pages", document various parts of the operating system for users and developers. They are one of the two parts of the SerenityOS documentation.
+
+The other part of the SerenityOS documentation is the developer documentation, which can be found in the `Documentation` folder in the repository ([online link](https://github.com/SerenityOS/serenity/tree/master/Documentation)). This documentation is focused on setting up a SerenityOS installation and workflow, as well as contributing to its development.
+
+Note that documentation might cover standardized topics (such as standard POSIX C library functions) or SerenityOS-specific extensions (such as custom file formats). There is always the possiblity of SerenityOS being non-spec-compliant, even if that is not documented.
```

Comment:
Not sure about the second sentence here. Perhaps an aspirational statement?

`SerenityOS intends to be spec-complaint with industry standard specifications. Non-compliance of with particular specification should be documented in the relevant sections.`

Reply:
Yeah, I just wanted to have this small word of warning that even though we provide POSIX stuff, we're non-compliant in sometimes obvious, sometimes subtle ways.